README.md

```diff
@@ -25,6 +25,7 @@ Add a `.env` file to the root of the project with the following variables:
 ```text
 HMPPS_AUTH_USERNAME= # A valid HMPPS Auth Username
 HMPPS_AUTH_PASSWORD= # A valid HMPPS Auth Password
+NOTIFY_API_KEY= # An API key for GOV.UK Notify
```

Comment:
Is the answer to "How do I get a/the `NOTIFY_API_KEY`?" just "ask someone/in Slack"?

Reply:
Yeah, I reckon so, we don't have a single place for this stuff (yet)